src/js/relay.firefox.com/installation_indicator.js

```diff
@@ -15,7 +15,10 @@
       const localLabels = localRandomAliasCache
         .filter(address => address.description.length > 0)
         .map(address => ({
-          type: "random",
+          // `type` can be removed as soon as the website is updated to watch
+          // for mask_type
```

Comment:
This is the update referred to: https://github.com/mozilla/fx-private-relay/pull/1990